app/account/forms.py

```diff
@@ -112,3 +115,38 @@ class ChangeEmailForm(Form):
     def validate_email(self, field):
         if User.query.filter_by(email=field.data).first():
             raise ValidationError('Email already registered.')
+
+
+class EditProfileForm(Form):
+    first_name = StringField('First name', validators=[
+        DataRequired(),
+        Length(1, 64)
+    ])
+    last_name = StringField('Last name', validators=[
+        DataRequired(),
+        Length(1, 64)
+    ])
+    ''' This will not be hardcoded in the future,
+        should be populated on the backend
+        and changeable by admin '''
+    affiliation_options = [
+        ('1', 'Veteran'),
+        ('2', 'Active Duty'),
+        ('3', 'National Guard'),
+        ('4', 'Reservist'),
+        ('5', 'Spouse'),
+        ('6', 'Dependent'),
+        ('7', 'Family Member'),
+        ('7', 'Supporter'),
+        ('8', 'Other')]
+    affiliation = SelectMultipleField(
+        'Affiliation',
+        choices=affiliation_options)
+    about_me = TextAreaField('About Me')
```

Comment:
- Do we want this to be required?
- Is this the same as `bio` in the `User` model, if so, should the variable be called bio?